src/services/travis.service.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable import/prefer-default-export,no-await-in-loop */
+import fetch from 'node-fetch';
+
+function mapTravisBuildToEndorBuild(item) {
+  const formattedBuild = {};
+  formattedBuild.id = item.id;
+  formattedBuild.buildNumber = item.number;
+  formattedBuild.state = item.state;
+  formattedBuild.previousState = item.previous_state;
+  formattedBuild.duration = item.duration;
+  formattedBuild.type = item.event_type;
+  formattedBuild.user = item.created_by.login;
+  formattedBuild.started_at = item.started_at;
+  formattedBuild.finished_at = item.finished_at;
+  return formattedBuild;
```

Comment:
Not a big deal, but I'm curious why you didn't just return an object:

```javascript
return {
  id: item.id,
  buildNumber: item.number,
  ...
};
```

It would be cleaner I think.

Reply:
Sure, personal preference I suppose.